entity-framework/core/managing-schemas/scaffolding/index.md

```diff
@@ -2,88 +2,259 @@
 title: Reverse Engineering - EF Core
 description: Reverse engineering a model from an existing database using Entity Framework Core
 author: bricelam
-ms.date: 11/13/2018
+ms.date: 03/24/2023
 uid: core/managing-schemas/scaffolding
 ---
-# Reverse Engineering
+# Scaffolding (Reverse Engineering)
```

Comment:
Should it be qualified to distinguish from ASP.NET Scaffolding?

Reply:
Will add a NOTE box.